sys/src/9/ufs/ufs_harvey.h

```diff
@@ -120,16 +99,21 @@ typedef enum vtype Vtype;
  * It may be unnecessary - if so, we should use inodes directly.
  */
 typedef struct vnode {
+	vnode		*next;
+	vnode		*prev;
+
+	Ref		ref;		/* Refcount */
+	QLock		v_vnlock;	/* u pointer to vnode lock */
+
 	/*
 	 * Fields which define the identity of the vnode.  These fields are
 	 * owned by the filesystem (XXX: and vgone() ?)
 	 */
-	const char 	*v_tag;			/* u type of underlying data */
 	inode		*v_data;
 	MountPoint	*v_mount;
```

Comment:
Would that be CamelCase starting with a capital for struct fields?  I've seen some examples of fields starting with a capital letter, but wasn't sure if it was an aberration...  I'd usually prefer starting with lower case (camelCase) but always prefer to be consistent at least.

The v_* fields were copied directly, but I'm feeling more relaxed about renaming things at the vnode level or above - the v_ bit is redundant.